components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -2514,3 +2540,323 @@ def object_table_query(request, objtype, objid, conn=None, **kwargs):
     tableData['parentId'] = ann['parentId']
     tableData['addedOn'] = ann['addedOn']
     return tableData
+
+
+def build_url(request, name, api_version, **kwargs):
```

Comment:
Unfortunately, if we don't return a full url here, then we can't offer a 'browsable' API in the browser, since links in the JSON would not be 'clickable'. This feature would be really nice to have since it makes the API much more user-friendly (E.g. like https://api.github.com/users/will-moore)